ocaml/xapi/xapi.ml

```diff
@@ -1030,11 +1030,14 @@ let server_init () =
           ; ("Initialise TLS verification", [], init_tls_verification)
           ; ("Running startup check", [], startup_check)
           ; ("Registering SMAPIv1 plugins", [Startup.OnlyMaster], Sm.register)
+          ; ( "Initialising SMAPIv1 state"
+            , []
+            , Storage_smapiv1_wrapper.initialise
+            )
```

Comment:
I am not sure if it would cause any issues, why are these two initialisation tasks swapped in order?